common.yml

```diff
@@ -815,6 +815,8 @@ functions:
           set -v
           set -e
           mkdir -p bin
+          ${_set_shell_env}
+          go mod vendor -v
```

Comment:
> what did you end up deciding re: how you wanted to handle the vendoring for this PR?

@rychipman, I took @huan-Mongo's suggestion to temporarily revendor on Evergreen to ensure that static checks and compilation pass.

I'll remove these two lines before merging and subsequently push the vendored changes directly to master.